lighthouse-cli/index.js

```diff
@@ -113,9 +121,9 @@ if (cli.listAllAudits) {
 }
 
 if (cli.listTraceCategories) {
-  const categories = lighthouse.traceCategories;
+  const traceCategories = lighthouse.traceCategories;
 
-  process.stdout.write(JSON.stringify({traceCategories: categories}));
+  process.stdout.write(JSON.stringify({traceCategories}));
```

Comment:
I thought the use for this was to print the trace categories as input into ChromeDriver. Was the plan not to get that list from the CLI?